csharp/HOLMS.Platform/HOLMS.Platform/Support/DTOBuilders/Booking/Genesis/ChannelRushReservationRequestBuilder.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using HOLMS.Platform.Support.ReservationTags;
+using HOLMS.Support.Time;
+using HOLMS.Types.Booking.Channels;
+using HOLMS.Types.Booking.Reservations;
+using HOLMS.Types.CRM.Guests;
+using HOLMS.Types.Supply.RoomTypes;
+
+namespace HOLMS.Platform.Support.DTOBuilders.Booking.Genesis {
+    public class ChannelRushReservationRequestBuilder {
```

Comment:
I've become less and less of a fan of builder classes. I've found our builders hard to remember to maintain and use in practice. What would be your opinion of static factory methods on a partial class peer of GenesisReservationRequest?

This feels a bit like "Mandatory Compliance is Optional."